skbio/stats/ordination/_principal_coordinate_analysis.py

```diff
@@ -62,37 +84,60 @@ def pcoa(distance_matrix):
 
     Notes
     -----
-    It is sometimes known as metric multidimensional scaling or
-    classical scaling.
-
-    .. note::
-
-       If the distance is not euclidean (for example if it is a
-       semimetric and the triangle inequality doesn't hold),
-       negative eigenvalues can appear. There are different ways
-       to deal with that problem (see Legendre & Legendre 1998, \S
-       9.2.3), but none are currently implemented here.
-
-       However, a warning is raised whenever negative eigenvalues
-       appear, allowing the user to decide if they can be safely
-       ignored.
+    Principal Coordinate Analysis is sometimes known as
+    metric multidimensional scaling or classical scaling.
+
+    If the distance is not euclidean (for example if it is a
```

Comment:
This portion should be left as it was before i.e. under the `.. note::` directive. The only thing that should have been removed from the `.. note::` directive is the sentence "Principal coordinate analysis ....".

Reply:
ah, thanks for clarifying